lucille-core/src/test/java/com/kmwllc/lucille/core/HeartbeatTest.java

```diff
@@ -48,6 +57,18 @@ public void testWatcher() throws Exception {
 
     String lastLine = lines.skip(currentLineCount - 1).findFirst().get();
     assertTrue(lastLine.contains("INFO Heartbeat: Issuing heartbeat"));
+
+    //Thread.sleep(1000);
+
+    Collection<Thread> nonSystemThreadsAfter =
+        ThreadUtils.findThreads(t -> !ThreadUtils.getSystemThreadGroup().equals(t.getThreadGroup()));
+
+    for (Thread thread : nonSystemThreadsAfter) {
+      log.info("THREAD testWatcher AFTER: {}", thread.getName());
+      if (ThreadNameUtils.isLucilleThread(thread)) {
+        log.info("testWatcher failed here");
```

Comment:
we're logging a message but how are we triggering an actual test failure here?

Reply:
resolved in [cdc333a](https://github.com/kmwtechnology/lucille/pull/172/commits/cdc333a75b671f998894dc5e830c75e13bb5819d)